scripts/release-unpublish.js

```diff
@@ -42,14 +47,22 @@ const main = async () => {
     );
     log('The following versions will be deleted:');
     log(versionsToDelete.map((v) => `   - ${v}`).join('\n'));
-    versionsToDelete.map(async (versionToDelete) => {
-      const { stderr } = await exec(`npm unpublish ${packageName}@${versionToDelete} ${dryRun}`);
-      if (stderr) {
-        log(`❌ Error ${packageName}@${versionToDelete} not deleted!\n`);
-      } else {
-        log(`✅ Package ${packageName}@${versionToDelete} deleted!\n`);
+
+    for (const versionToDelete of versionsToDelete) {
+      try {
```

Comment:
In case that happens, does it mean some versions haven't been deleted? If so, what should one do if they notice an error like this? Maybe we can have some tips for dealing with corner case scenarios.

Reply:
Makes sense, thanks.

For now, printing the error plus having simple tips should suffice.